ml-proto/spec/eval.ml

```diff
@@ -289,7 +289,7 @@ and eval_hostop c hostop vs at =
      * Since we currently only support i32, just test that. *)
     if I64.gt_u new_size (Int64.of_int32 Int32.max_int) then
       Trap.error at "memory size exceeds implementation limit";
-    Memory.grow mem delta;
+    Memory.grow mem (Int64.div delta Memory.page_size);
```

Comment:
Hm, I wonder if we should adjust the grow_memory operator as well. But probably for a separate patch.